### PR TITLE
Don't mark empty backticks as inline code

### PR DESCRIPTION
**Pull request by parlough**
Also removes what appears to be a duplicate pattern entry for this match. Let me know if it exists for a reason.

### CODE REVIEW

File: test/test_files/comments.dart [hunk 1]
```diff
@@ -11,6 +11,12 @@
 /// ...
 var a;
 
+/// ``
+var noInlineCode;
+
+/// ` `
+var inlineCode;
+
```
Comment (DanTup) [inline test/test_files/comments.dart:19]: Apparently we don't ahve anye xisting tests for this with non-whitespace contents - perhaps it's worth adding one (because it's not obvious from this test whether whitespace might be special rather than just treated the same as other content).

Comment (parlough) [reply test/test_files/comments.dart]: Good point! I added an inline code span golden with more than just whitespace :)
